### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by samschan-ms**
Added EXO journal FAQ. @KCCross to review

### CODE REVIEW

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -176,8 +176,18 @@ sections:
       - question: |
           Can I automatically remove encryption on incoming and outgoing mail?
         answer: |
-          Admins can set up a mail flow rule to remove encryption for outgoing mail. You can only set up a rule to remove encryption for incoming mail that originates from your organization.
+          Admins can set up a mail flow rule to remove encryption for outgoing mail. You can only set up a rule to remove encryption for incoming mail that originates from your Exchange Online organization.
 
+      - question: |
+          Can I automatically remove encryption to journal mail?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          Can I automatically remove encryption on journal mail?
```

File: microsoft-365/compliance/ome-faq.yml [hunk 2]
```diff
@@ -176,8 +176,18 @@ sections:
...
+        answer: |
+          For an Exchange Online mailbox, admins must enable journal decryption and set up an Exchange Online journaling rule to generate a decrypted copy of the mail into the journaling mailbox. The journaling rule will take any mail and/or attachments that have encryption and send the oringial plus a decrypted copy into the journal mailbox. You can only set up a journaling rule that can decrypt mail or attachment when the encrypted item originates from your Exchange Online organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          For an Exchange Online mailbox, admins must enable journal decryption and set up an Exchange Online journaling rule to generate a decrypted copy of the mail into the journaling mailbox. The journaling rule will take any mail or attachment that have encryption and send the original plus a decrypted copy into the journaling mailbox. You can only set up a journaling rule that can decrypt mail or attachments when the encrypted item originates from your Exchange Online organization.  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

File: microsoft-365/compliance/ome-faq.yml [hunk 3]
```diff
@@ -176,8 +176,18 @@ sections:
       - question: |
           Can I automatically remove encryption on incoming and outgoing mail?
         answer: |
-          Admins can set up a mail flow rule to remove encryption for outgoing mail. You can only set up a rule to remove encryption for incoming mail that originates from your organization.
+          Admins can set up a mail flow rule to remove encryption for outgoing mail. You can only set up a rule to remove encryption for incoming mail that originates from your Exchange Online organization.
```
Comment (KCCross) [inline microsoft-365/compliance/ome-faq.yml:204]: Remove Exchange Online here. The organization is M365. The tenant is Azure.

**Comment by samschan-ms** [comment]
@KCCross ok. go ahead. looks good now.

**Comment by KCCross** [comment]
Made requested corrections.

**Comment by yogkumgit** [comment]
@tiburd  as KC has made corrections can it be merged?

**Pull request merged**